File: src/targets/javascript/jquery/fixtures/headers.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/headers',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'GET',
  headers: {
    accept: 'application/json',
    'x-foo': 'Bar',
<<<<<<< HEAD
    'x-bar': 'Foo'
=======
    'quoted-value': '"quoted" \'string\''
>>>>>>> upstream/master
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/java/asynchttp/fixtures/headers.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("GET", "https://httpbin.org/headers")
  .setHeader("accept", "application/json")
  .setHeader("x-foo", "Bar")
<<<<<<< HEAD
  .setHeader("x-bar", "Foo")
=======
  .setHeader("quoted-value", "\"quoted\" 'string'")
>>>>>>> upstream/master
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();